_attributes/common-attributes.adoc

```diff
@@ -58,7 +58,7 @@ endif::[]
 :descheduler-operator: Kube Descheduler Operator
 // Backup and restore
 :velero-domain: velero.io
-:velero-version: 1.12
+:velero-version: 1.14.0
```

Comment:
```suggestion
:velero-version: 1.14
```
Setting this to `1.14.0` creates a URL that 404s in the intro of https://76841--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.

From https://github.com/openshift/openshift-docs/blob/main/backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc?plain=1#L11

```
The OADP Operator installs link:https://{velero-domain}/docs/v{velero-version}/[Velero {velero-version}].
```

Reply:
@adellape, thanks for sharing this. I've fixed it